types/forge-viewer/index.d.ts

```diff
@@ -5,6 +5,7 @@
 //                 Jan Liska <https://github.com/liskaj>
 //                 Petr Broz <https://github.com/petrbroz>
 //                 Cyrille Fauvel <https://github.com/cyrillef>
+//                 Kean Walmsley <https://github.com/keanw>
```

Comment:
Do you want to be owner? This means approving PRs from others and sometime reacting on feedback from DT community. Note you can still submit PR w/o being owner. Also note that according to DT rules owner can't approve own changes.

Reply:
No no - I'm not sure why that was included in the request. Fat fingers, probably.

Reply:
Ok, then just remove it and I'll approve.

Reply:
Done!